src/dashboard/src/components/api/views.py

```diff
@@ -582,6 +565,8 @@ def path_metadata(request):
         return helpers.json_response(body, status_code=201)
 
 
+# TODO should this have auth?
+@_api_endpoint(expected_methods=['GET'])
 def processing_configuration(request, name):
```

Comment:
I think it makes sense to leave the odd return types as they are for now. As nice as a consistent set of return bodies is, this stage is more about making it code-consistent and documenting current behaviour. And for this endpoint in particular, it doesn't make sense to return JSON containing an XML body.